cypress/utils/data_utils.ts

```diff
@@ -1,21 +1,29 @@
 import * as faker from "faker";
 
-export function getStakeholderName(): string {
+export function getFullName(): string {
     return faker.name.findName();
```

Comment:
any reason to go with `FullName`. Why aren't we simply calling it as getName like Email?

Reply:
Because if you read the faker module documentation, this function specifically return a full name and not just first name or last name

Reply:
Hi @nitishSr I completely agree with you. Yes, Faker returns Full Name.
But 
1. We should not have a concern with what faker returns. I can see that, Any control entity name can be integer values too. It means we are in the state where **Name can be anything**. So If I create a function(getName) which returns two random words then this will be also useful for name value.
2. We are using these functions in the test case while creating control entity objects. Also, `FullName` will not be useful in each entity(stakeholder, stakeholder groups, business service, job functions, tags). As per the UI, I can see it demands **name** only; which should be more than 3 chars and less than 120 chars long.
@sshveta @nitishSr 
- So IMO, we should go with `getName` to make it more readable and generic while accessing in the test cases of all control entities.

Reply:
But this function returns full name and not any integer or alphanumeric value. The name conveys what the function does. It may not be useful anywhere, but it is specific to stakeholder name. Hence, this change is not required !